subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_df.c

```diff
@@ -7,6 +7,7 @@
 #include <stdint.h>
 #include <errno.h>
 #include <devicetree.h>
+#include <drivers/gpio.h>
```

Comment:
How much of CPU use does GPIO API's use in ISR add to Radio IRQ? Remember, Radio ISR (and LLL context) is to run in Zero-Latency ISRs. Please consider use of nRFx functions unless the caller context is Thread.

Reply:
> How much of CPU use does GPIO API's use in ISR add to Radio IRQ? Remember, Radio ISR (and LLL context) is to run in Zero-Latency ISRs. Please consider use of nRFx functions unless the caller context is Thread.

@cvinayak 
GPIO peripheral is used only by the function: `radio_df_ant_switching_gpios_cfg()` to configure pins selected for antenna switching. This function is called once in `lll_df_init()`. There is no direct use of GPIO peripheral by the code in ISR.

I'll change use of Zephyr GPIO API to nrfx api.